catalog/core/catalog-core-api/src/main/java/ddf/catalog/filter/FilterDelegate.java

```diff
@@ -130,7 +130,8 @@ public T exclude() {
      * @see FilterDelegate#propertyIsEqualTo(String, Object)
      */
     public T propertyIsEqualTo(String propertyName, String literal, boolean isCaseSensitive) {
-        return propertyIsEqualTo(propertyName, (Object) literal);
+        throw new UnsupportedOperationException(
```

Comment:
For all of these exceptions, I'd like to see a static string format: "%s not supported by org.opengis.filter.Filter Delegate." that you use in each throw clause.